module4/projects/mod4Project/retro.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Retro Guide
+---
+
+During projects, it is important that you meet with your partner or your team to discuss the progress of your project, as well as, at the end of the project reflecting on its completion. These retros provide an opportunity to acknowledge what is working well so you can consciously continue doing what works. It is also important to recognize what is not working or needs improvement, so those issues can be addressed and the whole team can collaborate on a solution.
+
+Below are questions that everyone should address together as a team. If you have a gist where you've been keeping your feedback notes, include notes from your discussion there. Otherwise, there are no deliverables for this activity. 
```

Comment:
Have we talked about gists before this? 

I think it might actually be helpful to have students send us a summary of their retro. I think it's good practice to take notes from a retro and I want to make sure they take thoughtful time on this.

I'm also thinking about how there are lots of instances where we ask teams to take notes. Maybe we should be very clear that they should be rotating each time and everyone should have a roughly equal time spent as notetaker?

Reply:
What do you think about creating a private slack channel for each of the groups and all instructors and that could be where they share their stand-up notes, retro notes, and other links.